### DISCUSSION TITLE
How to specify a MariaDB port?

### DISCUSSION THREAD

**Discussion by FreshImmuc** [postalserver/postal#1933] [Help with using Postal]
How do I specify a MariaDB port and not use the default 3006 port?

**Comment by willpower232** [comment]
Its in the config file

https://github.com/postalserver/postal/blob/9f0697f194209f5fae5e451ba8fb888413fe37fa/config/postal.defaults.yml#L28

and 

https://github.com/postalserver/postal/blob/9f0697f194209f5fae5e451ba8fb888413fe37fa/config/postal.defaults.yml#L45
